tests/test_logger.py

```diff
@@ -0,0 +1,105 @@
+from collections import defaultdict
+import os.path as osp
+
+import stable_baselines.logger as sb_logger
+
+import imitation.util.logger as logger
+
+
+def _read_csv_lines(lines) -> dict:
```

Comment:
why are we rolling our own rather than using https://docs.python.org/3/library/csv.html ?

Reply:
Switched to using python's csv library. thanks